status/status_test.go

```diff
@@ -354,6 +354,9 @@ func (s) TestStatus_ErrorDetails(t *testing.T) {
 			t.Fatalf("(%v).WithDetails(%+v) failed: %v", str(s), tc.details, err)
 		}
 		details := s.Details()
+		if len(details) != len(tc.details) {
+			t.Fatalf("len(s.Details()) = %v, want = %v.", len(details), len(tc.details))
```

Comment:
nit: We should use %d to format integers as this helps linters catch data type mismatches.

Reply:
I copied this missing check from the other test, so I updated it in both places.